hr_employee_calendar_planning/models/hr_employee.py

```diff
@@ -50,10 +50,40 @@ def _regenerate_calendar(self):
         two_weeks = bool(
             self.calendar_ids.mapped("calendar_id").filtered("two_weeks_calendar")
         )
-        if (
-            not self.resource_id.calendar_id
-            or not self.resource_id.calendar_id.auto_generate
-        ):
+        if self.resource_id.calendar_id.auto_generate:
+            self.resource_calendar_id.attendance_ids.unlink()
+            self.resource_calendar_id.two_weeks_calendar = two_weeks
+        seq = 0
+        for week in {"0", "1"} if two_weeks else {"0"}:
```

Comment:
Sorry, I didn't remember that sets don't respect order:

```suggestion
        for week in ["0", "1"] if two_weeks else ["0"]:
```